test/test-servers/hapi_scaffold_server.ts

```diff
@@ -51,7 +52,7 @@ server.route({
   },
 });
 
-server.register({register: errorHandler.hapi}, err => {
+server.register({register: errorHandler.hapi} as {} as hapi.PluginFunction<{}>, err => {
```

Comment:
Does this need to happen as part of this PR?

Reply:
Yes.  With the change to use `imports`, the types don't exactly align.  I will create an issue to track this.